apps/src/netsim/NetSimLogEntry.js

```diff
@@ -23,12 +23,16 @@ var Packet = require('./Packet');
 var netsimNodeFactory = require('./netsimNodeFactory');
 var dataConverters = require('./dataConverters');
 var formatBinary = dataConverters.formatBinary;
+var base64ToBinary = dataConverters.base64ToBinary;
+var binaryToBase64 = dataConverters.binaryToBase64;
 var BITS_PER_BYTE = require('./netsimConstants').BITS_PER_BYTE;
 
 /**
  * @typedef {Object} logEntryRow
  * @property {number} nodeID
- * @property {string} binary
+ * @property {base64Payload} base64Binary - base64-encoded binary
```

Comment:
base64Payload -> string

this part of the JSDoc should either be a class or a JS data type. see http://usejsdoc.org/tags-type.html